storage/bucket.go

```diff
@@ -1455,7 +1450,14 @@ func toRawLifecycle(l Lifecycle) *raw.BucketLifecycle {
 			},
 		}
 
-		setAgeCondition(r.Condition.AgeInDays, &rr.Condition.Age)
+		if r.Condition.AgeInDays > 0 {
+			rr.Condition.Age = googleapi.Int64(r.Condition.AgeInDays)
+		}
+
+		if r.Condition.AllObjects {
```

Comment:
Question: we may want to return an error here when both `AgeInDays` and `AllObjects` are set at the same time, what do y'all think?

Reply:
Discussed offline; AllObjects should take precedence if both are set.